cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/CompileCommand.java

```diff
@@ -263,7 +263,7 @@ public String getName() {
 
     @Override
     public void printLongDesc(StringBuilder out) {
-        out.append("Compiles Ballerina module(s) and generates shareable .balo file(s). \n");
+        out.append("compile Ballerina module(s) and generates shareable .balo file(s). \n");
```

Comment:
Compiles has been changed but not generates?

Reply:
Changed to "generate".